tests/unit/scene/test_scene_dunder_methods.py

```diff
@@ -0,0 +1,65 @@
+import arcade
+import pytest
+
+
+def test_len():
+    scene = arcade.Scene()
+    scene.add_sprite_list("Player")
+
+    assert len(scene) == 1
+
+    scene.add_sprite_list("Walls")
+    scene.add_sprite_list("Coins")
+
+    assert len(scene) == 3
+
```

Comment:
For best style, add an extra line here. Aside from imports and constant groupings, first-level module members conventionally have two lines of space between them.